sktools/src/sktools/scripts/skexp.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+'''Returns STO exponents according to the usual geometric series.'''
+
+import argparse
+import numpy as np
+
+from sktools.skdef import Skdef
+from sktools import skgen
+import sktools.common as sc
```

Comment:
```suggestion
```

Reply:
Any reason to make suggestions to your own PR, instead of just a separate PR/force push on top of this one?